Automated Release Notes (#2306)
gradio-app/gradio: .github/PULL_REQUEST_TEMPLATE.md

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have added a short summary of my change to the CHANGELOG in website/CHANGELOG.md
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# A note about the CHANGELOG

Hello 👋 and thank you for contributing to Gradio!

> All pull requests must update the change log located in website/CHANGELOG.md unless they are labeled with the no-changelog-update label.

Reviewer: Is `website/CHANGELOG.md` the correct path? I don't see this file

Reviewer 2: no you're right, @freddyaboulton this should point to the releases dir now. good catch!

> Please add a brief summary of the change to the Full Changelog section of the UPCOMING.md file and include

Reviewer: ```suggestion
Please add a brief summary of the change to the Full Changelog section of the CHANGELOG.md file and include
```

For example, "* Added a cool new feature :pr:123". 

If you would like to elaborate on your change further, feel free to include a longer explanation in the other sections.

> If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file directly on the 

Reviewer: ```suggestion
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
```

> Github web UI since that lets you upload files directly via drag-and-drop.

Reviewer: Looks great! Perhaps add a link to the release history on the website so that people know what kind of images / gifs they can include in the changelog?

Reviewer: Consider adding this information to CONTRIBUTING.md as well so contributors are not "surprised" when they open a PR

Reviewer 2: @abidlabs not sure what you mean in the first comment. They should be able to add anything that would render in markdown. Ideally we should advise them to use the github ui to edit the markdown file so that it automatically hosts images/gifs/videos and manages size etc, and so they don't end up adding the images to the repo or anything like that. We can still add a link to website release history anyway though.

Reviewer 2: nvm looks like @freddyaboulton already added the comment about the github ui!

Reviewer 2: Although, we should probably force images/media to be added that way. We should not allow them to host it in the repo so it doesn't balloon in size

Author: How do we enforce this @aliabd ?

Reviewer 2: I'm not sure

Reviewer: No worries, ignore my first comment.

> If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file directly on the 
> Github web UI since that lets you upload files directly via drag-and-drop.

Reviewer: ```suggestion
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
GitHub web UI since that lets you upload files directly via drag-and-drop.
```
